### PR TITLE
Examining a window gives tips for construction/deconstruction

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Examining a window now gives hints to the appropriate tool for the
next stage of construction/deconstruction.
/:cl:

- Also adds an EXCITING NEW DESCRIPTION to reinforced windows

### CODE REVIEW

**Comment by Tacolizard** [comment]
FEATURE FREEZE IS DEAD

**Comment by coiax** [comment]
Kor said it was fine. ALSO THE FEATURE FREEZE IS DEAD.

**Comment by Iamgoofball** [comment]
this needs to be standardized somehow, this is atrocious.

**Comment by RemieRichards** [comment]
it can be standardised with cyberboss' construction refactor.

**Comment by coiax** [comment]
Barring that, I don't think the code is particularly bad.

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -27,7 +27,23 @@
 
 /obj/structure/window/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>Alt-click to rotate it clockwise.</span>")
+	if(reinf)
+		if(anchored && state == WINDOW_SCREWED_TO_FRAME)
+			to_chat(user, "<span class='notice'>The window is <b>screwed</b> to the frame.</span>")
```
Comment (ChangelingRain) [inline code/game/objects/structures/window.dm]: Maybe replace all these `The window` strings with `[src]`?

Comment (coiax) [inline code/game/objects/structures/window.dm]: Well it's talking about... the window inside the frame unit? The tool use messages don't use [src] either, should I change that? I'm not entirely clear of what the window object actually represents when its reinforced.

**Comment by coiax** [comment]
@KorPhaeron is there any problem with this?

**Comment by Niknakflak** [comment]
Oh god, hand holding oh my god spoon feeding holy shot oh no ahaaaa

**Pull request merged**